.github/workflows/test_changed_notebooks.yml

```diff
@@ -0,0 +1,61 @@
+name: Test Jupyter Notebooks
+
+on:
+  workflow_dispatch: 
+  push:
+    branches:
+      - main
+    paths:
+      - '**.ipynb'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '**.ipynb'
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    container:
+      image: us-central1-docker.pkg.dev/ucb-datahub-2018/user-images/data100-user-image:4497642d
+      options: --user root
+      credentials:
+        username: _json_key
+        password: ${{ secrets.GCP_SERVICE_ACCOUNT_KEY }}
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: Get changed files
+      id: changed-files-git
+      run: |
+        cd ${GITHUB_WORKSPACE}
+        ls -a
+        git status || true
```

Comment:
git doesn't behave properly when the files are owned by a different user than the current one. I think that is why it kept failing. You might try running `chown -R root .` right before running `git status`.

Reply:
Fixed it. Thanks